packngo.go

```diff
@@ -324,6 +324,9 @@ type requestDoer interface {
 // NewRequest inits a new http request with the proper headers
 func (c *Client) NewRequest(method, path string, body interface{}) (*retryablehttp.Request, error) {
 	// relative path to append to the endpoint url, no leading slash please
+	if path[0] == '/' {
```

Comment:
https://golang.org/pkg/strings/#TrimLeft might make this slightly more obvious to read, but this is well enough.

`// relative path to append to the endpoint url, no leading slash please`, that's a well placed comment.